src/components/UI/LoadingIndicator.js

```diff
@@ -9,9 +9,20 @@ const ProgressButtonStatusIndicator = styled.div`
 `;
 
 const LoadingIndicator = ({ loading, loadingPosition, timeout, ...rest }) => (
-  <ProgressButtonStatusIndicator {...rest}>
+  <ProgressButtonStatusIndicator
```

Comment:
Added a few accessibility attribues, so we'd be able to use more accessible queries in tests